JeanAbayo/WeConnect-Api
# User can create an account

**Description**

- A user should be able to create an account

**How to test this locally**

- Ensure you have the project set locally
- `npm install` and after completion `npm start`
- Using your favorite Api surfer, mine is Postman, browse to localhost:8080/api/auth/register and use pass in a Json object of the format ☟
 ```
{
	"username": "Secretadum",
	"email": "secretoadm@test.com",
	"password": "secret",
	"confirm_password": "secret"
}
```
**Relevant Pivotal Tracker stories**

#155546061
